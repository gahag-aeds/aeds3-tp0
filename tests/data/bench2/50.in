50 25
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 
Sub 22 37
Min 2 17
Sub 24 39
Max 24 39
Add 29 44
Max 30 45
Sub 18 33
Sum 17 32
Add 27 42
Max 7 22
Sub 18 33
Sum 21 36
Sub 16 31
Sum 24 39
Sub 12 27
Min 25 40
Sub 4 19
Max 32 47
Sub 34 49
Sum 12 27
Add 17 32
Min 16 31
Add 29 44
Min 15 30
Add 25 40
