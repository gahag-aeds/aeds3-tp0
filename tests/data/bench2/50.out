10
92
82
823
88
768
857
6
83
680
7
7
